### PR TITLE
Adds d3-tick-format helper

### PR DESCRIPTION
**Pull request by thedig**

### CODE REVIEW

File: addon/helpers/d3-tick-format.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import { timeFormat } from 'd3-time-format';
+
+export function d3TickFormat([tickFormat]) {
```
Comment (ivanvanderbyl) [inline addon/helpers/d3-tick-format.js]: I don't think we need a specific tick format helper. The d3 format helpers are useful in many places. It'd be better to have `format` and `format-time` helpers which also work as tick formats.

Comment (zigahertz) [inline addon/helpers/d3-tick-format.js]: @ivanvanderbyl alright - here's an example of how we are thinking of using this helper:
https://github.com/LocusEnergy/ember-sparkles/pull/4/files#diff-33855cd461c3ef518666bb5cdea92472R9

are you proposing that we keep this file (`d3-tick-format`) but just rename it? The idea is that the axis helper needs to either take a string or a function for it's `tickFormat` argument

Comment (ivanvanderbyl) [inline addon/helpers/d3-tick-format.js]: That's an annoying problem, what about doing something like this https://github.com/ivanvanderbyl/maximum-plaid/blob/master/addon/helpers/format-fn.js

**Comment by zigahertz** [comment]
closing based on https://github.com/LocusEnergy/ember-sparkles/pull/14

File: addon/helpers/d3-tick-format.js (hunk 1 above)
Comment (zigahertz) [inline addon/helpers/d3-tick-format.js]: @ivanvanderbyl thanks - this is very nice, I decided to go a different direction for now, and use the `tickValue` method on the axis - you can see what I did here: https://github.com/LocusEnergy/ember-sparkles/blob/9e483374b45444ce1bc1b7ee284a540fbb2f9156/addon/helpers/ember-sparkles/axis.js#L13-L15

I found this idea here - http://bl.ocks.org/mbostock/3212294

Taras and I discussed this - we plan to handle this a bit more programmatically when we implement axes as contextual components (similarly to what you are doing in Maximum Plaid!)

**Pull request closed without merging**